src/soundanalysis.cs

```diff
@@ -123,6 +137,22 @@ interface SNClassifySoundRequest : SNRequest {
 
 		[Export ("initWithMLModel:error:")]
 		IntPtr Constructor (MLModel mlModel, [NullAllowed] out NSError error);
+
+		[Watch (8,0), TV (15,0), Mac (12,0), iOS (15,0), MacCatalyst (15,0)]
+		[Export ("initWithClassifierIdentifier:error:")]
+		IntPtr Constructor (string classifierIdentifier, [NullAllowed] out NSError error);
+
+		[Watch (8,0), TV (15,0), Mac (12,0), iOS (15,0), MacCatalyst (15,0)]
+		[Export ("knownClassifications", ArgumentSemantic.Copy)]
+		string[] KnownClassifications { get; }
+
+		[Watch (8,0), TV (15,0), Mac (12,0), iOS (15,0), MacCatalyst (15,0)]
+		[Export ("windowDuration", ArgumentSemantic.Assign)]
+		CMTime WindowDuration { get; set; }
```

Comment:
That won't work, as expected, with watchOS since you added the following on the top
```
#if WATCH
 using CMTimeRange = Foundation.NSObject;
 using CMTime = Foundation.NSObject;
```

Reply:
Okay there was a comment above it from 2 years ago that said remove the 
```
#if WATCH
 using CMTimeRange = Foundation.NSObject;
```
 when watch is enabled in CoreMedia, which the headers appear to be. So I removed this #if else.
 But I should check in xcode and make sure I can call CoreMedia in a watch project.

Reply:
Okay, looks like I can call CoreMedia in a watch project!